NExtends/Primitives/String.extensions.cs

```diff
@@ -93,7 +90,7 @@ public static bool isEmail(this string value)
 		{
 			try
 			{
-				MailAddress m = new MailAddress(value);
+				MailboxAddress m = new MailboxAddress(value);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused 'm' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1481)